gui/wxpython/nviz/mapwindow.py

```diff
@@ -1390,7 +1390,7 @@ def LoadDataLayers(self):
                 GError(parent=self, message=e.value)
             # when nviz.tools is not yet ready
             # during opening 3D view 2nd time
-            except:
+            except (AttributeError, TypeError):
```

Comment:
There is a lot of going on here, I would put general Exception to not break stuff.